examples/shared_samples/server_profile.rb

```diff
@@ -71,6 +71,7 @@
 item3.retrieve!
 puts "\nServer Profile updated successfully! Name: #{item3['name']}"
 
+# This method #get_availble_server is removed for  Oneview REST API Version 1600.
```

Comment:
better to say "This method supports till OneView REST API Version 1200" and add begin and rescue block for this endpoint

Reply:
@soodpr Updated.